server/sonar-db-dao/src/main/resources/org/sonar/db/component/ComponentMapper.xml

```diff
@@ -646,4 +646,14 @@
     DELETE FROM projects WHERE id=#{id,jdbcType=BIGINT}
   </delete>
 
+  <select id="selectComponentKeysHavingIssuesToMerge" resultType="Component">
+    SELECT DISTINCT p.KEE, p.UUID FROM PROJECTS p
+    JOIN ISSUES i
+      ON p.UUID = i.COMPONENT_UUID
+    JOIN PROJECT_BRANCHES b
+      ON i.PROJECT_UUID = b.UUID
+      AND b.MERGE_BRANCH_UUID = #{mergeBranchUuid,jdbcType=VARCHAR}
+      AND i.status != 'OPEN'
```

Comment:
Probably good enough for the POC, but maybe will need tuning.

Reply:
shouldn't this be something like "status == 'CONFIRMED' or (status =='OPEN' AND resolution not null)"? (to be tested with a DB)